pontoon/base/static/js/translate.js

```diff
@@ -1116,6 +1121,30 @@ var Pontoon = (function (my) {
     },
 
 
+    /*
+     * Push value(s) if it not exists in array
+     */
+    pushIfNotExist: function(array) {
+      for (var i=1, l=arguments.length; i<l; i++) {
+        if (array.indexOf(arguments[i]) === -1) {
+          array.push(arguments[i]);
+        }
+      }
+    },
+
+
+    /*
+     * Check if all specified values present in array
+     */
+    arrayInArray: function(array, values) {
+      var value = true;
+      for (var i=0, l=values.length; i<l; i++) {
+        value &= (array.indexOf(values[i]) !== -1) ? true : false;
+      }
+      return value;
+    },
+
+
```

Comment:
These two functions can be made more specific, e.g. `applyUntranslatedFilter` and `untranslatedFilterApplied`. Both should be passed only one argument - `filter.status`. And `['missing', 'fuzzy', 'suggested']` should be externalized as e.g. `getUntranslatedFilters` and reused. It will help save some lines of code.